test/core/end2end/fuzzers/api_fuzzer.proto

```diff
@@ -20,12 +20,10 @@ message Empty {}
 
 message ByteSlice {
     bytes value = 1;
-    bool intern = 2;
```

Comment:
yeah probably safer not to.... it wouldn't be terrible (some corpora entries would fail to parse, but would presumably be built up again in days/weeks)